utilities/openemr-cmd/openemr-cmd

```diff
@@ -294,15 +347,16 @@ FINAL_EXIT_CODE=0
 
 # Confirm the docker install or not.
 DOCKER_CODE=16
-which docker &>/dev/null
-[ $? -ne 0 ] && echo "Please check docker install or not." && exit $DOCKER_CODE
+if ! command -v docker &>/dev/null; then
+  echo "Please check docker install or not."
+  exit "${DOCKER_CODE}"
+fi
 
 # Script usage.
-if [ $# -eq 0 ] || [ "$FIRST_ARG" == '--help' ] || [ "$FIRST_ARG" == '-h' ]
-then
+if [[ $# -eq 0 || "${FIRST_ARG}" = '--help' || "${FIRST_ARG}" = '-h' ]]; then
   echo ""
-  echo "Usage: `basename $0` COMMAND [ARGS]"
-  echo "Usage: `basename $0` -d <docker name> COMMAND [ARGS]"
+  echo "Usage: $(basename "$0") COMMAND [ARGS]"
+  echo "Usage: $(basename "$0") -d <docker name> COMMAND [ARGS]"
```

Comment:
```suggestion
  echo "Usage: ${0##*/} COMMAND [ARGS]"
  echo "Usage: ${0##*/} -d <docker name> COMMAND [ARGS]"
```

…to avoid the subshell and avoid doing it twice :)

Reply:
I tend to forget about this way to grab the basename, nice.